behave_django/management/commands/behave.py

```diff
@@ -136,7 +153,9 @@ def handle(self, *args, **options):
                        k, v in
                        options.items() if k in passthru_args and v is not None}
 
-        if options['dry_run'] or options['use_existing_database']:
+        if options['runner_class']:
+            django_test_runner = options['runner_class'](**runner_args)
```

Comment:
Can't we simply use this line in the `else` clause and have `BehaviorDrivenTestRunner` set as a default in the `--runner-class` parser argument? :bulb: